### ISSUE TITLE
Windows Compile Clean Typo in Example 1

### ISSUE THREAD

**Issue by kykrueger** [ThrowTheSwitch/Unity#224]
In the MakeFile of Example 1, it looks like there is a typo in the OS detection.

```
elseifeq ($(OSTYPE),msys)
    CLEANUP = rm -f
        MKDIR = mkdir -p
        TARGET_EXTENSION=.exe
```

The line with CLEANUP is not tabbed, and instead uses spaces. Quite sure this may stop the make-file from cleaning up on the windows platform. 

Did this typo get carried anywhere? I am going to make a pull request to fix this.

**Comment by kykrueger** [comment]
I have made a pull request to fix this issue #225 .

**Comment by kykrueger** [comment]
Pull request was granted
